go/src/koding/kites/kloud/provider/marathon/testdata/multi-container.json.golden

```diff
@@ -98,7 +111,7 @@
           "health_check": [
             {
               "command": {
-                "value": "curl -f -X GET http://$$HOST:$${PORT_56789}/kite"
+                "value": "true"
```

Comment:
just a question, given the fact that i am not familiar with marathon,  why did we swicth from curl statement to true?